<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleSimulator</class>
 <widget class="QWidget" name="ConsoleSimulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>415</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>2</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="commandLog"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLineEdit" name="commandLine"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="bExecute">
     <property name="text">
      <string>Execute</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="bClearLog">
     <property name="text">
      <string>Clear log</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="bClear">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="bPasteLast">
     <property name="text">
      <string>Paste last</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
